openeo/rest/datacube.py

```diff
@@ -259,6 +260,36 @@ def load_disk_collection(cls, connection: Connection, file_format: str, glob_pat
         )
         return cls(graph=pg, connection=connection)
 
+    @classmethod
+    def load_stac(
+        cls,
+        url: str,
+        spatial_extent: Union[Dict[str, float], Parameter, None] = None,
+        temporal_extent: Union[Sequence[InputDate], Parameter, str, None] = None,
+        bands: Optional[List[str]] = None,
+        properties: Optional[Dict[str, Union[str, PGNode, Callable]]] = None,
+        connection: Connection = None,
```

Comment:
```suggestion
        connection: Optional[Connection] = None,
```

Reply:
True, updated in DataCube.load_collection too